clients/web/test/testEnv.jadehtml

```diff
@@ -13,7 +13,10 @@ html(lang="en")
     #g-global-info-apiroot.hide %HOST%/#{apiRoot}
     #g-global-info-staticroot.hide %HOST%/#{staticRoot}
 
-    each js in jsFiles
+    each js in jsFilesUncovered
+      script(src="#{js}")
```

Comment:
I would suggest we remove `libs.min.js` from line 8 and instead pass it in with `jsFilesUncovered`, just to increase the flexibility of this template slightly.